services/dy-2Dgraph/use-cases/cc/Dockerfile

```diff
@@ -54,22 +54,18 @@ LABEL simcore.service.settings='[{"name": "ports", "type": "int", "value": 8888}
 # -----------------------------------------------------------------------------
 FROM common AS specialised
 # set of arguments to copy the right notebook
-ARG NOTEBOOK_NAME=cc_oned.ipynb
-ARG NOTEBOOK_FOLDER_NAME=cc-oned
-# copy the notebook in the image
-COPY --chown=jovyan:users ${NOTEBOOK_FOLDER_NAME}/${NOTEBOOK_NAME} .
-# set jupyter to trust the notebook
-RUN jupyter trust ${NOTEBOOK_NAME}
+ARG NOTEBOOK_NAME
```

Comment:
Would be good to have a different naming conventions to distinguish between arguments and environment variables